gensim/models/word2vec.py

```diff
@@ -459,6 +459,7 @@ def __init__(
         self.total_train_time = 0
         self.sorted_vocab = sorted_vocab
         self.batch_words = batch_words
+        self.size = size
```

Comment:
okay I will remove [this line](https://github.com/markroxor/gensim/blob/4b27d3687971ea45bc950f877c1bb0fb208e4ca4/gensim/models/doc2vec.py#L616) instead.